Source/AwsCommonRuntimeKit/mqtt/Mqtt5Types.swift

```diff
@@ -442,21 +449,18 @@ extension ClientOperationQueueBehaviorType {
 
 /// Optional property describing a PUBLISH payload's format.
 /// Enum values match `MQTT5 spec <https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901111>`__ encoding values.
-public enum PayloadFormatIndicator {
+public enum PayloadFormatIndicator: Int {
```

Comment:
Might want to re-discuss the API with Bret after the discussion in the Ops meeting if this is a good idea.

Reply:
If it is, we should make a struct which contains data and format for the Payload instead of having two separate connected variables.

Reply:
I think this enum will be extremely rarely used or set in either direction and is fully separate from payload in all other language implementations. Generally, it'll be ignored by the customer. The payload is a Data type and that feels sufficient.

Reply:
Do we even need this indicator? We should just pass the bytes as it is.

Reply:
I don't think we need it since like you said, it'll always be bytes that can be converted to utf8 strings... but it's also part of the mqtt5 spec and expected to be there. It's possible someone may want to check its value for... something?